src/features/core/metrics/errorReporting/ErrorReportingService.js

```diff
@@ -0,0 +1,114 @@
+import { get } from '../../../../utils/utils';
+import { BugsnagProvider, ERROR_REPORT_PROVIDER_NAME_BUGSNAG } from './providers/Bugsnag';
+import { ERROR_REPORTING_SERVICE_GLOBAL_KEY_NAME } from '../../../../utils/constants';
+
+const DEFAULT_ERROR_REPORT_PROVIDER = ERROR_REPORT_PROVIDER_NAME_BUGSNAG;
+const AVAILABLE_ERROR_REPORT_PROVIDERS = [ERROR_REPORT_PROVIDER_NAME_BUGSNAG];
+const ERROR_REPORTS_ENABLED_CONFIG_KEY = 'statsCollection.errorReports.enabled';
+const ERROR_REPORTS_PROVIDER_NAME_CONFIG_KEY = 'statsCollection.errorReports.provider';
+
+// TODO: remove the '|| true' when we can get this config value from response
```

Comment:
[Note] Also, set the default value as `false`.

Reply:
Addressed this. Hence, resolving it.
CC https://github.com/rudderlabs/rudder-config-backend/pull/3027